tools/idf_monitor.py

```diff
@@ -288,8 +288,8 @@ def handle_key(self, key):
             self.serial_reader.stop()
         else:
             try:
-                key = self.translate_eol(key)
-                self.serial.write(codecs.encode(key))
+                key = self.translate_eol(bytes(u(""+key), 'utf-8'))
```

Comment:
The problem isn't u"abc", this works fine the same as in your output.

The problem is that u("abc") is an error in Python 2 & 3, as this function doesn't exist.

Reply:
So, for example, the way to write this is `bytes(u"" + str(key))` or `bytes(u"" + key)` if you know the type of key.